### PR TITLE
Inventory - Show channel/frequency information for radios

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Like ACE interaction actions, show radio channel/frequency information in the inventory UI item lists

### CODE REVIEW

File: addons/sys_gui/fnc_initDisplayInventory.sqf [hunk 1]
```diff
@@ -46,3 +46,53 @@ _ctrlContainerItems ctrlAddEventHandler ["MouseButtonUp", {
...
+                // Display frequency for single-channel radios (e.g. AN/PRC-77)
+                private _maxChannels = [_item, "getState", "channels"] call EFUNC(sys_data,dataEvent);
+
+                private _text = if (isNil "_maxChannels") then {
+                    private _txData = [_item, "getCurrentChannelData"] call EFUNC(sys_data,dataEvent);
+                    private _currentFreq = HASH_GET(_txData, "frequencyTX");
+                    format ["%1 %2 MHz", _name, _currentFreq];
+                } else {
+                    format [LELSTRING(ace_interact,channelShort), _name, _item call EFUNC(api,getRadioChannel)]
+                };
```
Comment (jonpas) [inline addons/sys_gui/fnc_initDisplayInventory.sqf]: We could put that in a core function as both ACE Interact and this uses the exact same logic.

File: addons/sys_gui/fnc_initDisplayInventory.sqf [hunk 2]
```diff
@@ -46,3 +46,53 @@ _ctrlContainerItems ctrlAddEventHandler ["MouseButtonUp", {
...
+
+                // Display radio keys in front of bound radios
+                // Case-insentive since PTT API function returns radio class in lower case while
+                // inventory display provides it in config case
+                private _radioKey = (_pttAssign findIf {_x == _item}) + 1;
+
+                if (_radioKey > 0 && {_radioKey < 4}) then {
+                    _text = format ["%1: %2", _radioKey, _text];
+                };
```
Comment (jonpas) [inline addons/sys_gui/fnc_initDisplayInventory.sqf]: Same for this, could take `_radioKey` as an argument.

**Review by jonpas — APPROVED** [review]
Looks good and can be merged despite the reviews (can be done at a later time).

**Comment by BrettMayson** [comment]
This method could replace what I was trying in https://github.com/CBATeam/CBA_A3/pull/1329, if this works better than that. I couldn't get that PR to work on corpses

**Comment by mharis001** [comment]
Added a function to `sys_core` to remove the duplicated code. Also, use this now for the ACE Arsenal name changing.

**Pull request merged**